include/opt-sched/Scheduler/bb_spill.h

```diff
@@ -141,6 +143,7 @@ class BBWithSpill : public SchedRegion {
                     InstCount slotNum, EnumTreeNode *trgtNode);
   void SetSttcLwrBounds(EnumTreeNode *node);
   bool ChkInstLglty(SchedInstruction *inst);
+  bool needsSLIL();
```

Comment:
```suggestion
  bool needsSLIL() override;
```

(or with my other suggestion, `const override`)

Reply:
When I added the override specifier to this function It started printing out a lot of warnings that said that the use of override was inconsistent, since other functions in BBWithSpill could have override but don't.  I have no objection to someone going through and making all the functions use override, but I don't want to do this in this PR which involves the ACO algorithm.